tests/test_operation.py

```diff
@@ -301,7 +301,7 @@ def test_incorrect_grad_recipe_length(self):
 
         class DummyOp(qml.operation.CVOperation):
             r"""Dummy custom operation"""
-            num_wires = 1
+            num_wires = 2
```

Comment:
Swapping the order of the validation caused a bug to be caught in these tests - they are called below with 2 wires, not 1!